### DISCUSSION TITLE
There seems to be some problem with branches 'updated' status

### DISCUSSION THREAD

**Discussion by authifyAlly** [community/community#82764] [Repositories]
### Select Topic Area

Bug

### Body

When opening branches link, the update status shows that branches are deleted, even for branches that are active.
![image](https://github.com/community/community/assets/117624289/2003f283-1368-48cd-ad1a-55760e108f06)

**Comment by ghostinhershell** [comment]
Welcome to the GitHub Community, @authifyAlly, we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category.

With that being said, is it possible that these branches were automatically deleted after the pull requests that are related to those branches were merged into your repository? These resources might help:
- [Deleting a Branch](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-and-deleting-branches-within-your-repository#deleting-a-branch)
- [Managing the automatic deletion of branches](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-the-automatic-deletion-of-branches)

If that doesn't help, I recommend reaching out to support. Typically, the best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

**Reply by authifyAlly** [reply]
Hi @ghostinhershell 
It isn't just my repo . In every repo, when opening the branches section, it shows deleted. The following screenshot is from the [github Docs repository](https://github.com/github/docs/branches). The last commit was 20 mins ago. You can notice even the main branch is shown as deleted.

![image](https://github.com/community/community/assets/117624289/4ca799c8-f9ff-443c-9d06-c794fa2786f7)
